shell/app/node_main.cc

```diff
@@ -116,28 +116,40 @@ int NodeMain(int argc, char* argv[]) {
     }
 
     node::LoadEnvironment(env);
+    v8::Isolate* isolate = env->isolate();
+
+    {
+      v8::SealHandleScope seal(isolate);
```

Comment:
TIL what a `SealHandleScope` is!

```
/**
 * A SealHandleScope acts like a handle scope in which no handle allocations
 * are allowed. It can be useful for debugging handle leaks.
 * Handles can be allocated within inner normal HandleScopes.
 */
```